### PR TITLE
📖 book: fix broken link to kustomize

### PR DESCRIPTION
**Pull request by sbueringer**
Signed-off-by: Stefan Büringer buringerst@vmware.com

**What this PR does / why we need it**:
Fixes broken link, this currently blocks all PRs.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

### CODE REVIEW

File: docs/book/src/developer/providers/implementers-guide/configure.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ patchesStrategicMerge
 - manager_config.yaml
 ```
 
-[kustomizeyaml]: https://github.com/kubernetes-sigs/kustomize/blob/master/docs/glossary.md#kustomization
+[kustomizeyaml]: https://kubectl.docs.kubernetes.io/references/kustomize/glossary/#kustomization
```
Comment (schrej) [inline docs/book/src/developer/providers/implementers-guide/configure.md:49]: Wouldn't https://kubectl.docs.kubernetes.io/references/kustomize/kustomization/ be better here?

Comment (sbueringer) [reply docs/book/src/developer/providers/implementers-guide/configure.md]: Thx, you're right. Done

**Review by killianmuldoon — APPROVED** [review]
/lgtm

**Review by fabriziopandini — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**